db/models.py

```diff
@@ -1,4 +1,8 @@
 from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
+
+import settings
```

Comment:
`from django.conf import settings` this is the correct way of importing setting in a project